src/fusor/examples/bcr_abl1.json

```diff
@@ -1,112 +1,133 @@
 {
   "type": "CategoricalFusion",
-  "structural_elements": [
-    {
-      "type": "TranscriptSegmentElement",
-      "transcript": "refseq:NM_004327.3",
-      "gene_descriptor": {
-        "type": "GeneDescriptor",
-        "id": "normalize.gene:BCR",
-        "gene_id": "hgnc:1014",
-        "label": "BCR"
-      },
-      "element_genomic_end": {
-        "id": "fusor.location_descriptor:NC_000022.11",
-        "type": "LocationDescriptor",
-        "label": "NC_000022.11",
-        "location": {
-          "type": "SequenceLocation",
-          "sequence_id": "refseq:NC_000022.11",
-          "interval": {
-            "type": "SequenceInterval",
-            "start": {
-              "type": "Number",
-              "value": 23253980
-            },
-            "end": {
-              "type": "Number",
-              "value": 23253981
+  "structure": {
```

Comment:
we should make sure to revert any changes in `/examples/` and have them handled separately in the @jarbesfeld PR

Reply:
will do! :D

Reply:
ended up changing these for this PR since they are needed for nomenclature tests, which I wanted to ensure still functioned properly after the model updates